apps/users/templatetags/userindicator.py

```diff
@@ -29,3 +29,8 @@ def get_next_url(request):
 def userindicator(context):
     context['redirect_field_value'] = get_next_url(context['request'])
     return context
+
+
+@register.filter(name="get_key_value")
+def get_key_value(some_dict, key):
+    return some_dict.get(key, '')
```

Comment:
So, we can remove this templatetag again, right?

Reply:
and yes that was my other attempt at adding it, forgot to remove, I can squash commit if you think it's fine to do it like this